packages/react-components/lib/reports/utils.ts

```diff
@@ -1,6 +1,6 @@
 import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
 
-export interface ConfigProps {
+export interface ReportConfigProps {
```

Comment:
It seems that this is named `*ConfigProps` because the various components are named `*Config`. Do you know what they are named like that? Their functions doesn't look like a config.

Reply:
I'm guessing since `xxxStateReport` is already used, the components wrapping around them can't use them same name and thus the `*Config` behind?